### PR TITLE
Data retrieval

### PR DESCRIPTION
**Pull request by pedrocamargo**
This is a really basic implementation of the function, but should attend

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -5,3 +5,5 @@ Shapely==1.7.1
 tqdm==4.60.0
 dataclasses==0.8
 pyshp==2.1.3
+sqlalchemy
```
Comment (Sleepingwell) [inline requirements.txt:8]: Out of interest, why do you need sqlalchemy?

Comment (pedrocamargo) [reply requirements.txt:8]: It is what I use to connect to the database where the indices are held (sqlite now, postgresql as soon as we move that way). Once the database is up, I can go ahead and process the data into month data/vehicle

Comment (pedrocamargo) [reply requirements.txt:8]: I should point that the Python code is temporary, as it reads the daily files...

File: cvts/settings.py [hunk 2]
```diff
@@ -116,6 +116,8 @@ def _get_path(var, must_exist=False):
 #: Path to Valhalla configuration file.
 VALHALLA_CONFIG_FILE = os.path.join(CONFIG_PATH, 'valhalla.json')
 
+DATA_LAKE = '/mnt/data_lake/'
```
Comment (Sleepingwell) [inline cvts/settings.py:119]: Can we change that to be configured from an environment variable in a similar way to other settings in this file?

Comment (pedrocamargo) [reply cvts/settings.py:119]: We can, but where are we setting up the variables?

File: cvts/_data_retrieval.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                'type':    'via'
+            }
+    """
+    dates = [datetime.fromtimestamp(epochtime) for epochtime in range(from_instant, to_instant + DAY, DAY)]
+    data = []
+    for date in dates:
+        fldr = join(DATA_LAKE, str(date.year), str(date.month).zfill(2), datetime.strftime(date, "%Y%m%d"))
+        if not isdir(fldr):
+            continue
+        file = join(fldr, f'vehicle_{vehicle_id}.gzip')
```
Comment (Sleepingwell) [inline cvts/_data_retrieval.py:40]: ... I didn't know that f'...' existed. Thanx!

**Review by Sleepingwell — COMMENTED** [review]
Happy to merge and I can deal with the integration if you are.

**Comment by pedrocamargo** [comment]
I am happy with that too.

File: cvts/settings.py (hunk 2 above)
Comment (Sleepingwell) [reply cvts/settings.py:119]: I do it in my .bash_local... not ideal and I have always intended to put a config file somewhere 'at the system level'

File: requirements.txt (hunk 1 above)
Comment (Sleepingwell) [reply requirements.txt:8]: OK.

**Review by Sleepingwell — APPROVED** [review]

**Pull request merged**